releases.yml

```diff
@@ -1,4 +1,57 @@
 releases:
+  rc.1:
+    assembly:
+      basis:
+        brew_event: 47964519
+        reference_releases:
+          aarch64: 4.12.0-0.nightly-arm64-2022-11-16-220715
+          ppc64le: 4.12.0-0.nightly-ppc64le-2022-11-16-233925
+          s390x: 4.12.0-0.nightly-s390x-2022-11-16-220639
+          x86_64: 4.12.0-0.nightly-2022-11-16-220533
+      group:
+        advisories:
+          rpm: 104600
+          image: 104601
+          extras: 104602
+          metadata: 104603
+        release_jira: ART-5203
+        upgrades: 4.11.11,4.11.12,4.11.13,4.11.14,4.11.16,4.12.0-rc.0
```

Comment:
ECs went to dev-preview channel and that replaced FCs so... it feels weird not having upgrades for them, but i think that's as intended.